instance/templates/instance/ansible/vars.yml

```diff
@@ -174,6 +174,9 @@ COMMON_USER_INFO:{% for github_username in github_username_list %}
 {% endwith %}
 
 # Workers
+EDXAPP_WORKERS:
+  lms: 3
+  cms: 2
```

Comment:
The naming of the Ansible variables is terrible. `EDXAPP_WORKER_DEFAULT_STOPWAITSECS` is about the Celery workers, while the one you added is about the Gunicorn workers.  I propse to introduce two different sections, with comments that make clear which workers are meant.

(As an aside, the variable to set the number of Celery workers is called `edxapp_workers`, so the same, just in lower case.)

Reply:
That makes me facepalm so hard. Will change it.

Reply:
2af57223a3fb15d500b22c32453ed361fea43c0b (squashed and pushed, to prevent multiple CircleCI builds)